packages/analytics/amplify_analytics_pinpoint/example/macos/Runner/DebugProfile.entitlements

```diff
@@ -10,5 +10,9 @@
 	<true/>
 	<key>com.apple.security.network.server</key>
 	<true/>
+	<key>keychain-access-groups</key>
```

Comment:
Should not be committed

Reply:
Hmm why not?  Without this change every time we want to run Analytics Example app we need to add this required permission in for SecureStorage..

Reply:
What we have done in most all example apps is override the credential storage to not require this. We can do the same for analytics. I don't think you need to update the example app in this PR. I would probably just remove this change and in a follow up PR, add the override for endpointInfoStore.

Here is what that looks like for Auth:
```dart
final secureStorage = AmplifySecureStorage(
  config: AmplifySecureStorageConfig(
    scope: 'auth',
    // FIXME: In your app, make sure to remove this line and set up
    /// Keychain Sharing in Xcode as described in the docs:
    /// https://docs.amplify.aws/lib/project-setup/platform-setup/q/platform/flutter/#enable-keychain
    // ignore: invalid_use_of_visible_for_testing_member
    macOSOptions: MacOSSecureStorageOptions(useDataProtection: false),
  ),
);
final authPlugin = AmplifyAuthCognito(credentialStorage: secureStorage);
```

Reply:
Nice thanks!